crypto/tbs/src/lib.rs

```diff
@@ -264,5 +264,13 @@ mod tests {
         let bsig = combine_valid_shares(shuffle_sigs, threshold);
         let sig = unblind_signature(bkey, bsig);
         assert!(verify(msg, sig, pk));
+
+        // Not enough signatures
+        for _ in 0..6 {
+            sigs.pop();
+        }
+        let bsig = combine_valid_shares(sigs.clone().into_iter(), threshold);
```

Comment:
Wait, why does this even work :anguished: I know we currently check combine-ability externally but it feels like giving less than `threshold` shares should be an error. But tbh the entire `tbs` interface needs an overhaul imo.

Reply:
Not sure if this `ExactSizeIterator` constraint is the right now ... but `points.len()` didn't work without it ...